.github/workflows/all_module_test.yml

```diff
@@ -0,0 +1,52 @@
+name: All Modules Import Test
+on:
+  pull_request:
+    types: [ opened, reopened, edited, ready_for_review ]
+  push:
+
+permissions:
+    id-token: write
+    contents: read
+
+env:
+    GITHUB_TOKEN: ${{ secrets.BUILDER_PAT_ENCODED }}
+
+jobs:
+  all-module-import-test: 
+    runs-on: ubuntu-latest 
+    strategy:
+      fail-fast: true
+
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@1f99358870fe1c846a3ccba386cc2b2246836776 # v2.2.1
+        with:
+          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+      
+      - uses: actions/checkout@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: ${{ secrets.GHACTION_AWS_REGION }}
+          role-to-assume: ${{ secrets.GHACTION_AWS_ROLE }}
+          role-session-name: ${{ secrets.GHACTION_AWS_SESSION_NAME }}
+
+     
+      - name: Install system dependencies
+        run: |
+          pip install shyaml
+          pip install --upgrade pip==22.2.2
+          sudo apt update
+          sudo apt install -y wget
+          sudo apt install -y awscli
+          aws s3 cp ${{ secrets.BUILD_REQUIREMENTS }} /tmp/requirements.txt
```

Comment:
- Need to populate this pip package whenever unskript package is released, probably when the docker release is built.
- Now, if you are adding a new credential, till that credential is released, the legos for that credential will keep failing check. So, we will have to have a new release with the legos check passing.

Reply:
Took care of this.